### DISCUSSION TITLE
Why my train loss is nan?

### DISCUSSION THREAD

**Discussion by thevelvetunderground** [lululxvi/deepxde#922] [Q&A]
![image](https://user-images.githubusercontent.com/91045087/191397248-4e828111-7c79-405a-8b41-a902cb216821.png)

Please tell me how can I solve this problem?

**Comment by MINE0126** [comment]
Can you provide more information about your question? Or go check out the Q&A "I failed to train the network or get the right solution, e.g., large training loss, unbalanced losses."
